GVFS/GVFS.UnitTests/Maintenance/LooseObjectStepTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.Maintenance;
+using GVFS.Common.Tracing;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Mock.FileSystem;
+using GVFS.UnitTests.Mock.Git;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace GVFS.UnitTests.Maintenance
+{
+    [TestFixture]
+    public class LooseObjectStepTests
+    {
+        [TestCase]
+        public void LooseObjectsIgnoreTimeRestriction()
+        {
+            this.TestSetup(DateTime.Now.ToString(), out MockTracer tracer, out MockGitProcess gitProcess, out GVFSContext context);
+
+            LooseObjectsStep step = new LooseObjectsStep(context, requireCacheLock: false, ignoreTimeRestriction: true);
+            step.Execute();
+
+            tracer.StartActivityTracer.RelatedErrorEvents.Count.ShouldEqual(0);
+            tracer.StartActivityTracer.RelatedWarningEvents.Count.ShouldEqual(0);
+            List<string> commands = gitProcess.CommandsRun;
+            commands.Count.ShouldEqual(1);
+            commands[0].ShouldEqual("prune-packed -q");

```

Comment:
It's great that we are checking the command we run. Is there any way to check the environment variables? Or, should we leave that to the functional tests?

Reply:
Unfortunately the env variables are harder to test.  The good news is that the functional tests definitely cover this case.  I found out the hard way :-).